python/ray/_private/worker.py

```diff
@@ -1226,7 +1226,11 @@ def init(
                 passed_kwargs[argument_name] = passed_value
         passed_kwargs.update(kwargs)
         builder._init_args(**passed_kwargs)
-        return builder.connect()
+        ctx = builder.connect()
+        from ray._private.usage import usage_lib
```

Comment:
Why do we import here btw? (why not at the top level?)

Reply:
There is some circular dependency issue.